.dependabot/config.yml

```
@@ -643,3 +643,9 @@ update_configs:
     - match:
         dependency_name: demisto/*
         update_type: all
+        
+  - package_manager: python
+    directory: /docker/dxl
+    update_schedule: live
+  - package_manager: docker
+    directory: /docker/dxl
```

You are missing the settings for schedule and automerge. The script should have added this automatically. Please modify:
```suggestion
    directory: /docker/dxl
    update_schedule: daily
    automerged_updates:
    - match:
        dependency_name: demisto/*
        update_type: all
```